cheroot/wsgi.py

```diff
@@ -276,6 +278,11 @@ def get_environ(self):
 
         return env
 
+    def _get_peer_uid(self):
+        creds = self.req.server.socket.getsockopt(socket.SOL_SOCKET, socket.SO_PEERCRED, struct.calcsize('3i'))
```

Comment:
There's no `SO_PEERCRED` in Python 2: https://utcc.utoronto.ca/~cks/space/blog/python/AbstractUnixSocketsAndPeercred